src/misc/lv_log.c

```diff
@@ -101,6 +101,7 @@ void _lv_log_add(lv_log_level_t level, const char * file, int line, const char *
                lvl_prefix[level], LOG_TIMESTAMP_EXPR func);
         vprintf(format, args);
         printf(LOG_FILE_LINE_FMT "\n" LOG_FILE_LINE_EXPR);
+        fflush(stdout);
```

Comment:
Could you please clarify if this is an intentional implementation or an inadvertent error?

Reply:
I was fighting a little to display the logs in the failing tests, but I don't remember if it was really required in the end. Anyway, I think it's useful to be sure the logs are always flushed.